### ISSUE TITLE
Focus on Textarea when opening a private message modal

### ISSUE THREAD

**Issue by crookedneighbor** [HabitRPG/habitica#4606]
Just making this an issue as a reminder to myself.

**Comment by excentris** [comment]
@crookedneighbor The fix seems merged, shouldn't this issue be closed?

**Comment by crookedneighbor** [comment]
Hmmm... seems to me that it's only working in the development environment. I can't get it to auto-focus on the textarea in production.

**Comment by crookedneighbor** [comment]
Figured out the reason. 

@TheHollidayInn The directive you wrote worked in development, but when it got  minified for production, the directive broke. The way angular does things is kinda funky, and breaks when minified unless the array syntax is used. 

Here's the commit where I fixed it: https://github.com/HabitRPG/habitrpg/commit/d28ed6badf6b584eda69078e04adca14bf8631d3

If you see other directives, controllers, services, etc that aren't using the array syntax, feel free to PR a fix, because in all likelihood, they are not working. :sweat:
